q2_boots/core_metrics.py

```diff
@@ -32,24 +33,71 @@ def core_metrics(ctx, table, sampling_depth, metadata,
 
     tables = bootstrapped_tables.values()
 
+    alpha_metric_names = alpha_metrics
+
+    alpha_metrics = {'PHYLO': {}, 'NONPHYLO': {}}
```

Comment:
Why do `PHYLO` and `NONPHYLO` map to dicts? It seems like we only ever use the values later and not the keys.